openupgradelib/openupgrade.py

```diff
@@ -2688,7 +2688,7 @@ def delete_record_translations(cr, module, xml_ids, field_list=None):
                 columns=", ".join(list_columns),
                 values=", ".join(
                     [
-                        '\'{{"en_US": {x} -> "en_US"}}\'::jsonb'.format(x=x)
+                        "jsonb_build_object('en_US', {x}) -> 'en_US'".format(x=x)
```

Comment:
```suggestion
                        "jsonb_build_object('en_US', {x} -> 'en_US')".format(x=x)
```

Reply:
Done, does this behavior like this ok
`2023-07-05 08:44:56,144 160720 DEBUG mig15_0001 OpenUpgrade: 1 rows affected after 0:00:00.000256 running 
                UPDATE mail_template
                SET (subject, report_name) = (jsonb_build_object('en_US', subject -> 'en_US'), jsonb_build_object('en_US', report_name -> 'en_US'))
                WHERE id = 3 `